lib/trusted-mpt-light-client-types/src/signed_data.rs

```diff
@@ -0,0 +1,30 @@
+use unionlabs::{
+    encoding::{Bincode, Encode},
+    primitives::{H256, H512},
+};
+
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "bincode", derive(bincode::Encode, bincode::Decode))]
+pub struct SignedData<T: Encode<Bincode>> {
+    data: T,
```

Comment:
fields should be pub since you derive serde

Reply:
we talked through this and decided to keep this private and document the reason